client/my-sites/upgrades/checkout-thank-you/header.jsx

```diff
@@ -68,6 +69,19 @@ const CheckoutThankYouHeader = React.createClass( {
 			} );
 		}
 
+		if ( isGuidedTransfer( this.props.primaryPurchase ) ) {
+			if ( typeof this.props.primaryPurchase.meta === 'string' ) {
```

Comment:
Just curious, What would you expect this to be if its not a string?
If you expect it to be null/blank then can we just do `if ( this.props.primaryPurchase.meta ) {`